File: Python/gestorAplicacion/Hija/mensajeNegocio.py
#Es un mensaje formal, tiene informacion sobre el negocio del usuario
class MensajeNegocio(Mensaje):
    #No permite crear mensajes con campos vacios
    def __init__(self, titulo, cuerpo, logo, descripcion, termino, nombreCompletoA, nombreCompletoB):
        super.__init__(titulo, cuerpo)
        self._logo = logo
        self._descripcion = descripcion
        self._termino = termino
        self._nombreCompletoA = nombreCompletoA
        self._nombreCompletoB = nombreCompletoB

    def __str__(self):
        return "MensajeNegocios [titulo=" + self.getTitulo() + ", cuerpo=" + self.getCuerpo() + ", logo=" + self._logo + ", descripcion=" + self._descripcion + ", termino=" + self._termino + ", nombreCompletoA=" + self._nombreCompletoA + ", nombreCompletoB=" + self._nombreCompletoB + "]"

    def getLogo(self):
        return self._logo

    def setLogo(self, logo):
        self._logo = logo
        
    def getDescripcion(self):
        return self._descripcion
        
    def setDescripcion(self, descripcion):
        self._descripcion = descripcion
        
    def getTermino(self):
        return self._termino
        
    def setTermino(self, termino):
        self._termino = termino
        
    def getNombreCompletoA(self):
        return self._nombreCompletoA
        
    def setNombreCompletoA(self, nombreCompletoA):
        self._nombreCompletoA = nombreCompletoA
        
    def getNombreCompletoB(self):
        return self._nombreCompletoB
        
    def setNombreCompletoB(self, nombreCompletoB):
        self._nombreCompletoB = nombreCompletoB

File: Java/src/gestorAplicacion/Hija/MensajeNegocio.java
package gestorAplicacion.Hija;

import gestorAplicacion.Padre.*;

//Es un mensaje formal, tiene informacion sobre el negocio del usuario
class MensajeNegocio extends Mensaje{
	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;
	private String logo;
	private String descripcion;
	private String termino;
	private String nombreCompletoA;
	private String nombreCompletoB;
	
	//No permite crear mensajes con campos vacios
	public MensajeNegocio(String titulo, String cuerpo, String logo, String descripcion, String termino,
			String nombreCompletoA, String nombreCompletoB) {
		super(titulo, cuerpo);
		this.logo = logo;
		this.descripcion = descripcion;
		this.termino = termino;
		this.nombreCompletoA = nombreCompletoA;
		this.nombreCompletoB = nombreCompletoB;
	}
	
	public String toString() {
		return  "MensajeNegocios [titulo=" + this.getTitulo() + ", cuerpo=" + this.getCuerpo() 
			+ ", logo=" + logo + ", descripcion=" + descripcion + ", termino=" + termino
			+ ", nombreCompletoA=" + nombreCompletoA + ", nombreCompletoB=" 
			+ nombreCompletoB + "]";
	}

	String getLogo() {
		return logo;
	}

	void setLogo(String logo) {
		this.logo = logo;
	}

	String getDescripcion() {
		return descripcion;
	}

	void setDescripcion(String descripcion) {
		this.descripcion = descripcion;
	}

	String getTermino() {
		return termino;
	}

	void setTermino(String termino) {
		this.termino = termino;
	}

	String getNombreCompletoA() {
		return nombreCompletoA;
	}

	void setNombreCompletoA(String nombreCompletoA) {
		this.nombreCompletoA = nombreCompletoA;
	}

	String getNombreCompletoB() {
		return nombreCompletoB;
	}

	void setNombreCompletoB(String nombreCompletoB) {
		this.nombreCompletoB = nombreCompletoB;
	}
	
}
